site/reference/fn/render/README.md

```diff
@@ -0,0 +1,63 @@
+---
+title: "`render`"
+linkTitle: "render"
+type: docs
+description: >
+   Render a package
+---
+<!--mdtogo:Short
+   Render a package.
+-->
+
+`render` executes the pipeline of functions defined in the Kptfile [kptfile reference].
```

Comment:
Try to be consistent with:
1) go/kpt-docs-style-guide
2) kpt.dev/book/ 

Examples:
- Kptfile -> `Kptfile`
- "sub packages" -> "subpackages"
etc.

I'l let you do a pass on this before further comments.